components/arm/arm.go

```diff
@@ -60,6 +62,10 @@ const (
 	SubtypeName = resource.SubtypeName("arm")
 )
 
+// MTPoob is a string that all MoveToPosition errors should contain if the method is called
+// and there are joints which are out of bounds.
+const MTPoob = "cartesian movements are not allowed when arm joints are out of bounds"
```

Comment:
what do we think of `MTPoob` as the name for this error?
Open to feedback!

Reply:
Personally I think the name `MTPoob` is hilarious and we should keep it :)